### ISSUE TITLE
#107 Rankng

### ISSUE THREAD

**Issue by SENOSuit** [HUNTERS1984/tenposs#15]
## Content

● ランキング形式の設定（ユーザー、サービス、商品など）

**Comment by SENOSuit** [comment]
### 藤吉さん

＊ランキングについてはユーザーへのインセンティブ（商品券、割引クーポン付与など）があったほうがアクティブに見てもらえると思いますのでその説明があったほうがいいですね。

### 酒井

ランキングについては正直悩んでいます。これがうまく働く仕様を検討したいです。

**Comment by ryo2mattock** [comment]
そうですね、ただランキングとしても運用するクライアントも継続できるような仕組みや展開イメージを我々がある程度想定し用意しておかないとダメです。
そして、ユーザーサイドでいうと面白いしインセンティブが得れるというものがあれば継続しますしアプリもですが。お店へのアクティブさも良い方向性に進みますのでエンゲージメントが向上します。

なので私としては、Instagramでの投稿に関連する企画でのアクティブさや。来店数などをベースにある程度こちらで企画を用意して提供で初期は良いと思います。

**Comment by SENOSuit** [comment]
appの方でも書きましたが、
ユーザーランキングなどが機能するのはそれなりにユーザー数があるサービスだと思うので、これがインセンティブとして働くかどうかは、解析しつつ見てみたいところではあります。

**Comment by ryo2mattock** [comment]
そうですね。これはver2での実装になりますね。

**Comment by SENOSuit** [comment]
こちらも一度ペンディングで。

**Comment by ryo2mattock** [comment]
はい。
